components/src/core/mobile-stepper/mobile-stepper.tsx

```diff
@@ -36,7 +37,10 @@ const makeStyles = (
             borderRadius: 8,
             marginHorizontal: 4,
             overflow: 'hidden',
-            backgroundColor: props.inactiveColor || Colors.gray['200'],
+            backgroundColor: props.inactiveColor || Colors.gray[200],
+        },
+        circleDark: {
+            backgroundColor: props.inactiveColor || theme.colors.disabled,
```

Comment:
Why introduce a new class for this instead of the logic you had in the regular circle class before?

Reply:
I was having issues with storybook knobs and tests. Was hoping this would fix it... no luck. I'll revert this and figure out what's really happening with the tests.

Reply:
Upon quick look at the output it appears the inactive color passed as a prop is not taking effect.